docs/content/PageLayout.mdx

```diff
@@ -172,25 +173,39 @@ See [storybook](https://primer.style/react/storybook?path=/story/layout-pagelayo
   <PropsTableRow
     name="divider"
     type={`| 'none'
-| 'line'`}
+| 'line'
+| {
+    narrow?:
+      | 'none'
+      | 'line' 
+      | 'filled'
+    regular?:
+      | 'none'
+      | 'line'
+    wide?:
+      | 'none'
+      | 'line'
+  }`}
```

Comment:
(not blocker) we should think about how we want to show "type" in the docs, showing the typescript type isn't the most readable 😅

<img width="742" alt="image" src="https://user-images.githubusercontent.com/1863771/181780397-abebf598-41ee-4899-b6e6-1e2268c3594e.png">

Reply:
Agreed! Open to ideas here.